docs/vsphere-golden-image.md

```diff
@@ -4,13 +4,15 @@ This guide describes the thought process of creating a Windows virtual machine b
 
 ## 1. Select a compatible Windows Server version
 
-Currently, the Windows Machine Config Operator (WMCO) stable version only supports Windows Server Semi-Annual
-Channel (SAC): Windows Server 2004, that includes the patch [KB4565351](https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351), 
-required by the operating system to allow using the [hybrid OVN Kubernetes networking with a 
-custom VXLAN port](setup-hybrid-OVNKubernetes-cluster.md#vSphere) feature.
-
-*Please note that Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809 cannot be used, since 
-the patch is not available.*
+Currently, the Windows Machine Config Operator (WMCO) stable version supports:
+* Windows Server 2022 (must contain the OS-level container networking patch [KB5012637](https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d))
+* Windows Server 20H2
+
+*Please note that Windows Server 2004 and Windows Server 2019, version 1909 (SAC) are no longer supported.
```

Comment:
I prefer to have only the supported version list which we have already.  We don't want to keep adding not supported versions down the line. It's extra work and redundant information.

Reply:
+1